R/util.R

```diff
@@ -378,6 +378,32 @@ util.rbind_union <- function(dfs){
 # util.rbind_union(dfs)
 # util.rbind_intersection(dfs)
 
+util.check_index_is_unique <- function(df, index_cols = NULL){
```

Comment:
I think the name of the function could be a little clearer. When you say TRUE in response to check_index_is_unique, that presumably means the index is unique. But couldn't it also mean that you _checked_ if the index is unique? I might rename to `util.index_is_unique` to remove this admittedly small amount of ambiguity.